.github/workflows/mirror_protoc_release.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror Releases
+on:
+  # Trigger manually in the UI
+  workflow_dispatch:
+  # Trigger daily at 06:10 UTC
+  schedule:
+    - cron: "10 6 * * *"
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          ./proto/private/protoc/mirror_protoc_releases.sh
+          npx @bazel/buildifier proto/private/protoc/versions.bzl
```

Comment:
Does this need to be a separate step or can it be part of the script?

Reply:
It could be part of the script. The choice of `npx` here is technically tied to the software we know to be installed on the GHA runner machine, so if someone ran `mirror_protoc_releases.sh` locally, it might not be installed. However I don't expect anyone to run it locally, so I'm happy to relocate it if you prefer.